docs/sources/mimir/release-notes/v2.8.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Grafana Mimir version 2.8 release notes"
+menuTitle: "V2.8 release notes"
+description: "Release notes for Grafana Mimir version 2.8"
+weight: 300
+---
+
+# Grafana Mimir version 2.8 release notes
+
+Grafana Labs is excited to announce version 2.8 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Changed default value of block storage retention period** The default value for `-blocks-storage.tsdb.retention-period` was `24h` and now is `13h`.
+- **Query-frontend cached results now contain timestamp** This allows Mimir to check if cached results are still valid based on current TTL configured for tenant. Results cached by previous Mimir version are used until they expire from cache, which can take up to 7 days. If you need to use per-tenant TTL sooner, please flush results cache manually.
```

Comment:
Is this one of the top four most important features in this release? Seems like an important change that improves correctness, but I wonder if there are more important changes.

Reply:
I will look again into the changelog. Those three are things that caught my eye. Please suggest if you have some points to place there.